src/wallet/rpcwallet.cpp

```diff
@@ -3373,35 +3392,38 @@ RPCHelpMan signrawtransactionwithwallet()
 static RPCHelpMan bumpfee_helper(std::string method_name)
 {
     bool want_psbt = method_name == "psbtbumpfee";
+    const std::string incremental_fee{CFeeRate(DEFAULT_MIN_RELAY_TX_FEE).ToString(FeeEstimateMode::SAT_VB)};
```

Comment:
DEFAULT_INCREMENTAL_RELAY_FEE, no?

Reply:
Agreed, this looks like an oversight.

Reply:
`#include <validation.h>` -> `#include <policy.h>` in rpcwallet.cpp with the change

Reply:
Done in #20426, thanks for the catch @MarcoFalke